docs/state.md

```diff
@@ -9,12 +9,12 @@ While Linux is usable on all T2 models, some features are limited due to the lac
 - USB
 - Keyboard
 - Camera
-- Wi-Fi
+- Wi-Fi (requires macOS firmware)
+- Bluetooth (requires macOS firmware)
```

Comment:
only for models with BCM4377 Wifi/Bt chip

Reply:
doesnt wifi require firmware on all systems?

Reply:
yea. It's bluetooth specifically that doesn't need it for most systems

Reply:
ah I wasn't completely sure what line you commented on